reactioncommerce/reaction
# Fix getSlug imports

Resolves #4696   
Impact: **minor**  
Type: **bugfix**

## Issue
Error during migration. See linked issue.

## Solution
When getSlug was moved, some imports were not updated. Updated them.

## Breaking changes
None

## Testing
1. Set DB migration version to 6 or lower and start the app, or start the app on a fresh database.
2. Verify that no errors occur during the migration on startup
